tests/Microsoft.IdentityModel.Protocol.Extensions.Tests/CrossTokenTests.cs

```diff
@@ -70,9 +70,46 @@ public void CrossToken_ValidateToken()
             JwtSecurityTokenHandler.InboundClaimFilter.Add("iss");
             JwtSecurityTokenHandler.InboundClaimFilter.Add("nbf");
 
+            SecurityKey x509SecurityKey = new X509SecurityKey(KeyingMaterial.DefaultCert_2048);
+            SigningCredentials signingCredentials = new SigningCredentials(x509SecurityKey, SecurityAlgorithms.RsaSha256Signature, SecurityAlgorithms.Sha256Digest);
+            SecurityTokenDescriptor descriptor = new SecurityTokenDescriptor
+            {
+                AppliesToAddress = IdentityUtilities.DefaultAudience,
+                SigningCredentials = signingCredentials,
+                Subject = IdentityUtilities.DefaultClaimsIdentity,
+                TokenIssuerName = IdentityUtilities.DefaultIssuer,
+                Lifetime = new System.IdentityModel.Protocols.WSTrust.Lifetime(DateTime.UtcNow, DateTime.UtcNow + TimeSpan.FromDays(1)),
+            };
+
+            TokenValidationParameters validationParameters = new TokenValidationParameters
+            {
+                AuthenticationType = IdentityUtilities.DefaultAuthenticationType,
+                IssuerSigningKey = x509SecurityKey,
+                ValidAudience = IdentityUtilities.DefaultAudience,
+                ValidIssuer = IdentityUtilities.DefaultIssuer,
+                ValidateIssuerSigningKey = true
+            };
+
+            // Test ValidateIssuerSigningKey = true, validationParameters.CertificateValidator == null, validationParameters.IssuerSigningKeyValidator == null
+            string jwtTokenTest = IdentityUtilities.CreateJwtToken(descriptor, jwtHandler);
+            ClaimsPrincipal jwtPrincipalTest = ValidateToken(jwtTokenTest, validationParameters, jwtHandler, ExpectedException.SecurityTokenValidationException("IDX10232:"));
```

Comment:
I don't think this method is the right place for these tests for SecurityKey validation. These tests are for ensuring Saml and Jwt tokens return the same ClaimsPrincipal. 

Maybe here:
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/dev4x/tests/System.IdentityModel.Tokens.Jwt.Tests/ValidatorsTests.cs#L141